.gitignore

```diff
@@ -0,0 +1,27 @@
+# General
```

Comment:
gitignore file is not required at the moment. Kindly remove it.

Reply:
Removed.